crates/swift-bridge-build/src/package.rs

```diff
@@ -18,6 +18,8 @@ pub struct CreatePackageConfig {
     pub out_dir: PathBuf,
     /// The name for the Swift package
     pub package_name: String,
+    /// Additional resources to copy into the package, first PathBuf is source and second is destination
+    pub resources: Vec<(PathBuf, PathBuf)>,
```

Comment:
Can we make the semantics of the destination more clear?
Right now it's hard to tell what the destination is.

 Maybe `pub resources: Vec<BundleResource>`

```rust
/// Describes a bundle resource to copy from some source path to some destination.
///
/// The source is a path to an existing file or directory that you wish to bundle.
///
/// The destination is a relative path that the resource will get copied to within the
/// generated Swift package's `Sources/YourLibraryName` directory.
///
/// # Examples
///
/// ```
/// # use crate::CopyBundleResourceDesc;
///
/// // If you are creating a Swift Package called "MyLibrary", this descriptor will
/// // lead to the package generator copying the `/path/to/images` directory to the
/// // the Swift package's `Sources/MyLibrary/assets/icons` directory.
/// let resource = CopyBundleResourceDesc::new("/path/to/images", "assets/icons");
/// ```
pub struct CopyBundleResourceDesc {
    source: PathBuf,
    destination: PathBuf,
}

impl CopyBundleResourceDesc {
    /// See [`CopyBundleResourceDesc`] for documentation.
    ///
    /// # Panics
    /// Panics if the destination is an absolute path.
    pub fn new(source: impl Into<PathBuf>, destination: impl Into<PathBuf>) -> Self {
        let destination = destination.into();
        assert!(destination.is_relative());
        Self {
            source: source.into(),
            destination,
        }
    }
}
```

Just a quick idea, open to other approaches.

Reply:
Ok, refactoring into CopyBundleResourceDesc done.